### DISCUSSION TITLE
Hide Body Part Mask Transfer

### DISCUSSION THREAD

**Discussion by NOSALIS** [soupday/cc3_blender_tools#18] [Ideas]
I would love to see masking done with the Hide Body tool in CC3 carry over with imports into Blender. Granted - this is isn't super hard to in Blender - but it's always nice to not have to repeat a step.

**Comment by soupday** [comment]
That information is, unfortunately, nowhere to be found in the export data. The best you can do is check the Delete hidden faces box on export from CC3/iClone.

**Reply by NOSALIS** [reply]
Smh I glossed over that feature because (with it being animation software) I interpreted that as "Human Face" and skipped it since I didn't know wtf it did. NOW I KNOW. 😄 Thanks for the tip!
